src/components/[guild]/CreateFormModal/components/FormCardEditable/components/FormFieldTitle.tsx

```diff
@@ -0,0 +1,19 @@
+import { Text, TextProps } from "@chakra-ui/react"
+import { CreateFieldParams, Field } from "components/[guild]/CreateFormModal/schemas"
+
+type Props = {
+  field: CreateFieldParams | Field
+} & TextProps
+
+const FormFieldTitle = ({ field, ...textProps }: Props) => (
```

Comment:
I really wanted to use a simple `FormLabel` instead, but we'd need to nest 2 `FormField`s in the `FormCardEditable` component, that's why I made a custom component here.